src/main/java/core/basesyntax/RandomColor.java

```diff
@@ -1,10 +1,19 @@
 package core.basesyntax;
 
+import java.util.function.Supplier;
+
 /**
  * <p>Имплементируйте необходимый функциональный интерфейс и его метод для случайного
  * выбора цвета</p>
  */
-public class RandomColor {
+public class RandomColor implements Supplier<String> {
     private static final String[] COLORS = {"red", "green", "blue", "yellow", "pink",
             "black", "white"};
+
+    @Override
+    public String get() {
+        int a = 0;
+        int b = 7;
+        return COLORS[a + (int) (Math.random() * b)];
```

Comment:
try to use new Random().nextInt()and don't create redundant variables

Reply:
Wow! Tanks! :)